src/requests/payment-request.ts

```diff
@@ -35,7 +35,13 @@ export class PaymentRequestClient {
 
 type PayNode = {
   type: ComponentType.Pay;
-  data: { fn: string | undefined };
+  data: {
+    fn: string | undefined;
```

Comment:
In the Editor, I believe `fn` is required for Pay - can it be here too?

Reply:
Good catch! Resolved by [755eef](https://github.com/theopensystemslab/planx-core/pull/334/commits/3755eef3715b99a71f74c7efc3c8a837ae7cc9ca)